src/main.ts

```diff
@@ -195,6 +196,31 @@ export default class ShikiPlugin extends Plugin {
 		}
 	}
 
+	registerInlineCodeProcessor(): void {
+		this.registerMarkdownPostProcessor(async (el, ctx) => {
+		    const inlineCodes = el.findAll(":not(pre) > code");
+		    for (let codeElm of inlineCodes) {
+		        let match = codeElm.textContent.match(/^\{([^\s]+)\} (.*)/i);  // format: `{lang} code`
+		        if (match && this.loadedLanguages.has(match[1])) {
+		            let tokens = await this.getHighlightTokens(match[2], match[1]);
+		            tokens = tokens.tokens.flat(1);
+		            if (!tokens.length) continue;
+		            codeElm.innerHTML = '';  // clear content
```

Comment:
prefer the `.empty()` method that the Obsidian API adds to the `HTMLElement` prototype.